tests/test_health_operations.py

```diff
@@ -0,0 +1,143 @@
+import json
+import logging
+import pytest
+
+from utils.utils import get_noobaa_health_status
+import noobaa_sa.exceptions as e
+
+log = logging.getLogger(__name__)
+
+
+class Test_health_operations:
+    def setup_prereqs(self, unique_resource_name, random_hex, account_manager, bucket_manager):
+        """
+            Create account and bucket for performing health operation
+        """
+        account_name = unique_resource_name(prefix="account")
+        access_key = random_hex()
+        secret_key = random_hex()
+        account_manager.create(account_name, access_key, secret_key)
+        account_manager.list()
+        bucket_name = unique_resource_name(prefix="bucket")
+        bucket_manager.create(account_name, bucket_name)
```

Comment:
Please consider using a class-scoped pytest fixture for this setup. This way you wouldn't need to include all these fixture arguments in each test case. Check [this example](https://github.com/red-hat-storage/ocs-ci/blob/1f24abc0a6e7864baa40e15d1ac32270b77d3647/tests/functional/object/mcg/test_log_based_bucket_replication.py#L67-L116) for reference.

Reply:
Fixed